src/store/channels/saga.ts

```diff
@@ -52,6 +54,11 @@ export function* openConversation(conversationId) {
 
   yield call(setActiveConversation, conversationId);
   yield spawn(markConversationAsRead, conversationId);
+
+  const isSidekickOpen = yield select(isSecondarySidekickOpenSelector);
+  if (isSidekickOpen) {
+    yield spawn(toggleIsSecondarySidekick);
+  }
```

Comment:
I think it might make sense to have a specific `closeSecondarySidekick` and then you can just call that from here. That keeps this method much more scannable (i.e., developers don't have to interpret the `if` statement at all. It's just, "oh, open the conversation, mark messages as read, close the sidekick")

Then, I also don't think `closeSecondarySidekick` has to actually check if it's open... you can just set the boolean to false. If it doesn't change, nothing happens.